crates/topos-tce-storage/src/validator/tables.rs

```diff
@@ -71,11 +76,38 @@ impl ValidatorPendingTables {
 
         let db = init_with_cfs(&path, default_options(), cfs)
             .unwrap_or_else(|_| panic!("Cannot open DB at {:?}", path));
+        let pending_pool = DBColumn::reopen(&db, cfs::PENDING_POOL);
+        let next_pending_id = {
+            let cf = pending_pool
+                .rocksdb
+                .cf_handle(cfs::PENDING_POOL)
+                .expect("Cannot get cf handle for pending pool");
+            let mut pending_iterator = pending_pool.rocksdb.raw_iterator_cf(&cf);
+
+            pending_iterator.seek_to_last();
```

Comment:
Is this expensive for a big DB? Or does rocksdb cache the last index?

Reply:
I'm not sure, but if I recall correctly, `rocksdb` based its `iterator` on an implicit snapshot of the data, it uses an ordered index to target the right key, I think.